site/content/community/users.md

```diff
@@ -10,12 +10,18 @@ considering their efforts before starting your own!
 
 ### General purpose plugins
 
-| Name           | Description                                                                                 |
-|:---------------|---------------------------------------------------------------------------------------------|
-| [go-plugin][2] | use gRPC to define WebAssembly plugins                                                      |
-| [waPC][5]      | uses an intuitive [IDL][6] and code generation to create WebAssembly plugins |
+| Name           | Description                                                                      |
+|:---------------|----------------------------------------------------------------------------------|
+| [go-plugin][2] | implements [Protocol Buffers][8] services with WebAssembly using code generation |
```

Comment:
reworded these as I noticed they were a little awkward, though otherwise unrelated to this PR.